index.tt

```diff
@@ -6,152 +6,251 @@
+          easier time setting their development environment.
+        </p>
+
+      </div>
+    </div>
+  </section>
+  <section class="tab-pane" id="build">
+    <div class="row-fluid">
+      <div class="span6">
+        <h2>Minimal docker image</h2>
+        <p>
+          Declarative way to build minimal docker images. No build tools inside
+          docker image, no complex multi stage build process, only what your
+          application needs.
+        </p>
+        <p>
+          The following Nix expression (<code>default.nix</code>) defines a
+          docker image with <strong>only</strong> Python 3 installed in it.
+        </p>
+        <pre class="well" style="margin-top:1em">
+{ pkgs ? import &lt;nixpkgs&gt; {}
+}:
+pkgs.dockerTools.buildLayeredImage {
+  name = "only-hello";
+  contents = [ pkgs.hello ];
+}</pre>
+        <p>To build and run the image you need to:</p>
+        <pre class="well" style="margin-top:1em">
+<span class="nix-command"><strong>nix-build</strong></span>
+...
+/nix/store/&#8230;-docker-image-only-hello.tar.gz
+<span class="nix-command">docker load -i ./result</span>
+...
+Loaded image: only-hello:fgzj21lg3hjv8bxlaabcsfjkh4fg5ssk
+<span class="nix-command">docker run only-hello hello"</span>
+Hello World</pre>
+        <p>
+          Learn more <a href="[%root%]nixpkgs/manual/#sec-pkgs-dockerTools">how
+            to build docker images</a>.
+        </p>
+
+      </div>
+      <div class="span6">
+
+        <h2>Declarative cloud images</h2>
+        <p>How hard would it be to build and configure a <strong>Amazon EC2</strong> image?</p>
+        <p>
+          With the following <code>amazon.nix</code> we defined nginx which is
+          serving example <code>/var/www</code> folder, having a valid ssl
+          certificate (via LetsEncrypt) and enabled recommended security
+          settings.
+        </p>
+        <pre class="well" style="margin-top:1em">
+{ pkgs, ...}:
+{
+  security.acme.acceptTerms = true;
+  security.acme.email = "nix@example.com";
+  services.nginx = {
+    enable = true;
+    recommendedGzipSettings = true;
+    recommendedOptimisation = true;
+    recommendedProxySettings = true;
+    recommendedTlsSettings = true;
+    virtualHosts."example.com" = {
+      enableACME = true;
+      forceSSL = true;
+      locations."/".root = "/var/www";
+    };
+  };
+}</pre>
+        <p>Now we just need to build it.</p>
+        <pre class="well" style="margin-top:1em">
+<span class="nix-command"><strong>nix-build '&lt;nixpkgs/nixos/release.nix&gt;' \</strong></span>
+<strong>    -A amazonImage.x86_64-linux \</strong>
+<strong>    --arg configuration ./amazon.nix \</strong>
+<strong>    -o ./result</strong>
+...
+<span class="nix-command">ls ./result/</span>
+nixos-amazon-image-20.09pre130979.gfedcba-x86_64-linux.vhd
+nix-support</pre>
+      </div>
+    </div>
+  </section>
 </div>
 
+<style type="text/css" media="screen">
```

Comment:
This will be moved to `css/nixos-site.css` before we close this PR. I just found it easier to work on this like this.